### PR TITLE
fix: default to latest retina tag as value for HELM_IMAGE_TAG

### PR DESCRIPTION
**Pull request by jimassa**
# Description

This PR dynamically pulls the latest tag from the repo and uses it as the default HELM_IMAGE_TAG.

## Related Issue

If this pull request is related to any issue, please mention it here. Additionally, make sure that the issue is assigned to you before submitting this pull request.

## Checklist

- [x] I have read the [contributing documentation](https://retina.sh/docs/contributing).
- [x] I signed and signed-off the commits (`git commit -S -s ...`). See [this documentation](https://docs.github.com/en/authentication/managing-commit-signature-verification/about-commit-signature-verification) on signing commits.
- [x] I have correctly attributed the author(s) of the code.
- [x] I have tested the changes locally.
- [x] I have followed the project's style guidelines.
- [x] I have updated the documentation, if necessary.
- [x] I have added tests, if applicable.

## Screenshots (if applicable) or Testing Completed

Please add any relevant screenshots or GIFs to showcase the changes made.

## Additional Notes

Add any additional notes or context about the pull request here.

---

Please refer to the [CONTRIBUTING.md](../CONTRIBUTING.md) file for more information on how to contribute to this project.

### CODE REVIEW

**Review by huntergregory — DISMISSED** [review]
Thanks for this fix!

**Comment by huntergregory** [comment]
Fixes #582

File: Makefile [hunk 1]
```diff
@@ -385,7 +385,10 @@ coverage: # Code coverage.
 manifests: 
 	cd crd && make manifests && make generate
 
-HELM_IMAGE_TAG ?= v0.0.2
+# Fetch the latest tag from the GitHub
+LATEST_TAG := $(shell curl -s https://api.github.com/repos/microsoft/retina/tags | jq -r '.[0].name')
```
Comment (timraymond) [inline Makefile:389]: Are these sensibly ordered chronologically? Would doing this with Releases provide a more predictable output?

Comment (jimassa) [reply Makefile]: True, updated it to use releases instead of tags

**Review by huntergregory — APPROVED** [review]

**Pull request merged**